hikari/internal/ux.py

```diff
@@ -26,6 +26,7 @@
 __all__: typing.List[str] = ["init_logging", "print_banner", "supports_color", "HikariVersion", "check_for_updates"]
 
 import importlib.resources
+import logging
 import logging.config
```

Comment:
This change is more a part of pyright compatibility from a static analysis perspective (although it might effect how pyright type interpret the types exported by the lib thus still effecting coverage) and pyright as a type checker explicitly ignores side effects from module loading thus both need to be explicitly imported.